avocado/core/safeloader.py

```diff
@@ -76,10 +76,10 @@ def get_docstring_directive(docstring):
     :type docstring: str
     """
     if docstring is None:
-        return None
-    result = AVOCADO_DOCSTRING_DIRECTIVE_RE.search(docstring)
+        return []
```

Comment:
I actually suggest we change the method name to `get_docstring_directives` since it now returns many directives.  Bonus points for `:rtype: list`